### ISSUE TITLE
ansible 2 doesn't like empty top-level 'vars' declarations

### ISSUE THREAD

**Issue by hfinucane** [ansible/ansible#11532]
More silly bugs, but this:

```

---

- hosts: all
  vars:
  gather_facts: no
  tasks:
  - debug: msg=foo
```

is fine in v1, but causes an error in v2:

```
ERROR! Vars in a playbook must be specified as a dictionary, or a list of dictionaries
```

**Comment by bcoca** [comment]
fixed via b5f3e84014f0c9fa88b5bd0ce5371d7306e22992

**Comment by hfinucane** [comment]
Thanks!
